CHANGES.txt

```diff
@@ -0,0 +1,99 @@
+Extension of Frangi filter to 3D data and other ridge detection filters
```

Comment:
Remove this file. List the new functionality and API changes in https://github.com/scikit-image/scikit-image/blob/master/doc/release/release_dev.rst